osbuild.spec

```diff
@@ -74,16 +74,16 @@ make man
 %py3_install
 
 mkdir -p %{buildroot}%{pkgdir}/stages
-install -p -m 0755 $(find stages/* -not -path "*/osbuild") %{buildroot}%{pkgdir}/stages/
+install -p -m 0755 $(find stages -type f) %{buildroot}%{pkgdir}/stages/
 
 mkdir -p %{buildroot}%{pkgdir}/assemblers
-install -p -m 0755 $(find assemblers/* -not -path "*/osbuild") %{buildroot}%{pkgdir}/assemblers/
+install -p -m 0755 $(find assemblers -type f) %{buildroot}%{pkgdir}/assemblers/
 
 mkdir -p %{buildroot}%{pkgdir}/runners
-install -p -m 0755 $(find runners/* -not -path "*/osbuild") %{buildroot}%{pkgdir}/runners
+install -p -m 0755 $(find runners -type f -or -type l) %{buildroot}%{pkgdir}/runners
 
 mkdir -p %{buildroot}%{pkgdir}/sources
-install -p -m 0755 $(find sources/* -not -path "*/osbuild") %{buildroot}%{pkgdir}/sources
+install -p -m 0755 $(find sources -type f) %{buildroot}%{pkgdir}/sources
```

Comment:
Why not drop the `find` entirely and copy the directories unchanged?

Reply:
Actually yeah, good idea.

Reply:
FTR, this would copy `__pycache__` and friends. Lets keep it the way it is.